android/library/src/main/res/raw/webcontent_default.css

```diff
@@ -0,0 +1,8 @@
+<style>
+    .MobileCSS { padding: 4%; width: 92%; }
```

Comment:
This is not a css file, we should load the css and then insert the tabs programatically, seems right?